admin/src/translations/pt-BR.json

```diff
@@ -0,0 +1,131 @@
+{
+    "plugin.name": "Comentários",
+    "plugin.section": "plugin de comentários",
+    "plugin.section.item": "Configurações",
+    "common.button.back": "Voltar",
+    "nav.header.moderation": "Moderação",
+    "nav.header.settings": "Configurações",
+    "nav.item.updates": "Updates",
+    "nav.item.discover": "Descobrir",
+    "nav.item.reports": "Reports",
+    "nav.item.settings": "Configurações",
+    "page.common.item.status.OPEN": "Abrir",
+    "page.common.item.status.BLOCKED": "Bloqueado",
+    "page.common.item.status.APPROVED": "Aprovedo",
+    "page.common.item.status.TO_REVIEW": "Para revisar ({count, number})",
+    "page.common.item.status.REJECTED": "Rejeitado",
+    "page.common.item.status.REMOVED": "Removido pelo autor",
```

Comment:
Thanks for contribution @rrfaria !

I've recognised that in base EN file there is a single translation missing:

```
"page.common.item.status.PENDING": "Pending"
```

Can you please add it also to your file? And I will be happy to merge it :)

Reply:
I added at same position as you did on yours